docs/getting_started.rst

```diff
@@ -372,25 +390,35 @@ Git Setup
 To work with ``git``, complete the following guides to set up ssh keys:
 https://help.github.com/articles/connecting-to-github-with-ssh/
 
-   .. note::
+**Note:**
+
+- ~~On Windows, you must always use ``Cygwin Terminal``, not ``Git Bash``~~.
+
+- ``Cygwin Terminal`` is now deprecated and no longer support, we are now using ``Git Bash``
+
+- (**DEPRECSTED**) If you have SSH configured and ``vagrant`` is still not working on Windows, you should add the ``export VAGRANT_PREFER_SYSTEM_BIN=true`` environment variable to the ``.bash_profile`` file, that helps you not add this variable on Cygwin repeatedly.!!
+
+  ..  code-block:: bash
+
+    $ cat >> ~/.bash_profile
+
+  Type ``export VAGRANT_PREFER_SYSTEM_BIN=true`` and press ``Ctrl + D``, then run:
 
-      - On Windows, you must always use ``Cygwin Terminal``, not ``Git Bash``.
+  ..  code-block:: bash
 
-      - If you have SSH configured and ``vagrant`` is still not working on Windows, you should add the
-        ``export VAGRANT_PREFER_SYSTEM_BIN=true`` environment variable to the ``.bash_profile`` file, that helps you
-        not add this variable on Cygwin repeatedly.
+    $ source ~/.bash_profile
 
-      ..  code-block:: bash
+Now, open the terminal and run the ``export`` command to check if the variable is added successfully
 
-          $ cat >> ~/.bash_profile
+Rsync with Git Bash
+-------------------
 
-      Type ``export VAGRANT_PREFER_SYSTEM_BIN=true`` and press ``Ctrl + D``, then run:
+To use rsync with `Git Bash`:
 
-       ..  code-block:: bash
+download [rsync-3.1](http://www2.futureware.at/~nickoe/msys2-mirror/msys/x86_64/rsync-3.1.2-2-x86_64.pkg.tar.xz) at http://www2.futureware.at/~nickoe/msys2-mirror/msys/x86_64/
```

Comment:
Should have the bullet for this section:
- Download [rsync-3.1](http://www2.futureware.at/~nickoe/msys2-mirror/msys/x86_64/rsync-3.1.2-2-x86_64.pkg.tar.xz) at http://www2.futureware.at/~nickoe/msys2-mirror/msys/x86_64/

- Extract and copy `rsync.exe` to `C:\Program Files\Git\usr\bin`, then re-open your terminal window.

Reply:
applied